lua/telescope/pickers/entry_display.lua

```diff
@@ -3,6 +3,76 @@ local strings = require "plenary.strings"
 local entry_display = {}
 entry_display.truncate = strings.truncate
 
+---@tag telescope.pickers.entry_display
+
+---@brief [[
+--- Entry Display takes care of formatting the text displayed for each entry.
+--- This includes adding highlight colors.
+---
+--- Most pickers will call the create function only once.
+--- The configuration.items passed in will provide the column
+--- widths which will normally be the same for every entry.
+--- This is why a single call is made to create.
+---
+--- The create function will use the column widths to create a generator table.
+--- Each item in the table is a function used to format the text in each column,
+--- in terms of width and how it's justified.
```

Comment:
I dont think we need to mention the generator table, because thats an internal thing. What comes out is a function that uses this generator tables (array) of functions